src/error.c

```diff
@@ -152,22 +152,39 @@ Obj FuncCURRENT_STATEMENT_LOCATION(Obj self, Obj context)
     return retlist;
 }
 
-Obj FuncPRINT_CURRENT_STATEMENT(Obj self, Obj context)
+Obj FuncPRINT_CURRENT_STATEMENT(Obj self, Obj stream, Obj context)
 {
     if (context == STATE(BottomLVars))
         return 0;
 
+    /* HACK: we want to redirect output */
+    /* Try to print the output to stream. Use *errout* as a fallback. */
+    if ((IsStringConv(stream) && !OpenOutput(CSTR_STRING(stream))) ||
+        (!IS_STRING(stream) && !OpenOutputStream(stream))) {
+        if (OpenOutput("*errout*")) {
+            Pr("PRINT_CURRENT_STATEMENT: can not open error stream\n", 0, 0);
+        }
+        else {
+            fputs("gap: panic, can not open *errout*!\n", stderr);
+            SyExit(1);
```

Comment:
These two lines could be changed to use the new `Panic` helper function. But that's not very important.

Reply:
Done.